ietf/meeting/models.py

```diff
@@ -266,9 +266,13 @@ def get_attendance(self):
             attended_per_meetingregistration | attended_per_meeting_attended
         ).distinct()
 
+        onsite=set(attended.filter(meetingregistration__meeting=self, meetingregistration__reg_type='onsite'))
+        remote=set(attended.filter(meetingregistration__meeting=self, meetingregistration__reg_type='remote'))
```

Comment:
Would it be better to do the exclusion in the query and just pull the counts over to Python? I'm thinking something like:
```
  onsite = attended.filter(...type='onsite').count()
  remote = attended.filter(...type='remote').exclude(...type='onsite').count()
```
(I'm not sure if that's an improvement.)

Reply:
I started there and it produced surprisingly wrong results. i have a dim memory that there is a bug in the stack that causes the sql query to fail - don't remember if the issue was in Django2 or in mysql. So I just avoided it for now, and I think the result is actually a little more readable.

Reply:
It does trade off doing things in django's memory rather than the databases, but this isn't a performance crunchpoint.